packages/osintgram/PKGBUILD

```diff
@@ -0,0 +1,49 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=osintgram
+pkgver=348.50d44fd
+pkgrel=1
+pkgdesc='OSINT tool on Instagram. It offers an interactive shell to perform analysis on Instagram account of any users by its nickname.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon')
+url='https://github.com/Datalux/Osintgram'
+license=('GPL3')
+depends=()
+makedepends=('git' 'python')
+source=("git+https://github.com/Datalux/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```

Comment:
As explained in [PKGBUILD-generic-git](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-generic-git#L23-L24), if [tag exists](https://github.com/Datalux/Osintgram/tags), prefer

```
git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
```

Don't forget to change `pkgver` then.

Reply:
Fixed